index.html

```diff
@@ -24,7 +24,7 @@ <h1>FREDDY K</h1>
         <section class="personal-info">
             <div class="personal-info-wrap">
                 <div class="personal-info-title">
-              <h2><a href="#" class="header-info">I'm Freddy Glad to see you!</a></h2>
+              <h1><a href="#" class="header-info">I'm Freddy Glad to see you!</a></h1>
```

Comment:
[optional] a tag is used to anchor links, based on UX it isn't necessary to wrap this heading with a tag.